service/src/main/resources/schemas/appsvc-metadata.schema.json

```diff
@@ -0,0 +1,156 @@
+{
+  "$schema": "https://json-schema.org/draft-07/schema",
+  "$id": "https://gitlab.cee.redhat.com/fix-mappings/schema/-/raw/main/appsvc-metadata.schema.json",
+  "title": "Application Services ET Manifest",
+  "description": "A temporary solution to attaching manifests and cve fix info to an erratum",
+  "type": "object",
+  "properties": {
+    "apiVersion": {
+      "description": "Version of the configuration file",
+      "enum": ["appsvcs.redhat.com/v2"]
+    },
+    "deliverables": {
+      "type": "array",
+      "minItems": 1,
+      "items": {
+        "anyOf": [
+          {
+            "$ref": "#/$defs/deliverables/pnc-build"
+          },
+          {
+            "$ref": "#/$defs/deliverables/pnc-analysis"
+          },
+          {
+            "$ref": "#/$defs/deliverables/pnc-operation"
+          }
+        ],
+        "properties": {
+          "type": {
+            "enum": ["pnc-build", "pnc-operation", "pnc-analysis"]
+          }
+        },
+        "required": ["type"]
+      },
+      "additionalProperties": false,
+      "additionalItems": false
+    },
+    "manifest": {
+      "type": "object",
+      "properties": {
+        "refs": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string"
+              },
+              "uri": {
+                "type": "string"
+              }
+            },
+            "required": ["type", "uri"]
+          },
+          "uniqueItems": true,
+          "minItems": 1
+        }
+      },
+      "required": ["refs"]
+    },
+    "simple-mapper": {
+      "type": "object",
+      "properties": {
+        "refs": {
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "type": {
+                "type": "string"
+              },
+              "version": {
+                "const": "0.0.2"
+              },
+              "fix": {
+                "$ref": "#/$defs/simple-mapper"
+              }
+            },
+            "required": ["type", "version", "fix"]
+          },
+          "minItems": 1
+        }
+      },
+      "required": ["refs"]
+    }
+  },
+  "oneOf": [{ "required": ["manifest"] }, { "required": ["deliverables"] }],
+  "additionalProperties": false,
+  "$defs": {
+    "deliverables": {
+      "pnc-build": {
+        "type": "object",
+        "properties": {
+          "type": {
+            "const": "pnc-build"
+          },
+          "buildId": {
+            "description": "PNC build identifier",
+            "type": "string"
+          }
+        },
+        "required": ["buildId", "type"],
```

Comment:
Could you please update the schema in fix-mappings/schema to match this?

Reply:
sure was on my list :-)